bundle/artifacts/expand_globs.go

```diff
@@ -30,6 +31,19 @@ func createGlobError(v dyn.Value, p dyn.Path, message string) diag.Diagnostic {
 }
 
 func expandGlobs(ctx context.Context, b *bundle.Bundle, name string) diag.Diagnostics {
+	err := b.Config.MarkMutatorEntry(ctx)
```

Comment:
Why not just make it a mutator again and rely on bundle.Apply? What's the benefit of having function like this now?

Reply:
good point, 240d4428b28756f5a131e00ca484124c483f78fc